scripts/link_obs.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# link BB.bin from a previously completed runs
+# $1 src_root_dir: the root directory of a previous automated simulation runs
+# $2 des_root_dir: the root directory of a clean install of simultion runs
+# $3 list_fault: a list to events to link
+
+# IMPORTANT: structure is for validation runs only.
+# TODO: translate into python and incorporate qcore.simluated_structure for flexibility
```

Comment:
Does this do the same as the above python script?

Reply:
i'll mark these as resolved for this PR. as its causing clutter.